src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Extensions/RoslynLspExtensions_SourceText.cs

```diff
@@ -30,6 +29,5 @@ public static TextChange GetTextChange(this SourceText text, TextEdit edit)
         => new(text.GetTextSpan(edit.Range), edit.NewText);
 
     public static TextEdit GetTextEdit(this SourceText text, TextChange change)
-        => RoslynLspFactory.CreateTextEdit(text.GetRange(change.Span), change.NewText.AssumeNotNull());
-
+        => RoslynLspFactory.CreateTextEdit(text.GetRange(change.Span), change.NewText ?? "");
```

Comment:
>"" [](http://example.com/codeflow?start=89&length=2)

Nit: string.Empty? not sure if it makes any difference...

Reply:
IIRC, it does make a difference, but an absurdly small one. :smile:

Reply:
I thought it didn't make a difference after .NET Framework 2.0 🤷‍♂️

Reply:
No, it definitely made a difference in .NET Framework 2.0. There was actually effort to inline `string.Empty` for empty string literals in the JIT more aggressively post .NET Framework 4.0. I know that because it caused a bug that @jasonmalinowski and I ran into where the value of `string.Empty` could become corrupted by the COM marshaller if strings were annotated incorrectly (e.g. `BStr` instead of `LPWStr`).

Crazy days.